src/index.html

```diff
@@ -5,10 +5,67 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <title>MOYO header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        >
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="*" class="nav__link is-active">
```

Comment:
all your links lead to 404 page( Rewrite href atribute in all links
```suggestion
            <a href="#Apple" class="nav__link is-active">
```